packages/eslint-plugin-svelte/src/configs/all.ts

```diff
@@ -6,7 +6,7 @@ export = {
 	extends: [baseExtend],
```

Comment:
Could this be addressed by specifying the type? It seems challenging to use `as const` extensively, so I thought this might be a simpler solution.

```diff
- export = {
+ import type { Linter } from 'eslint';
+ const config: Linter.Config = { ... };
+ export = config;
```

Reply:
Sure I can try that top !

Reply:
Didn't see most configs were generated, I've also fixed that